autopush/db.py

```diff
@@ -694,18 +724,25 @@ def update_last_message_read(self, uaid, timestamp):
 
 class Router(object):
     """Create a Router table abstraction on top of a DynamoDB Table object"""
-    def __init__(self, table, metrics, max_ttl=MAX_EXPIRY):
-        # type: (Table, IMetrics, int) -> None
+    def __init__(self, conf, metrics, max_ttl=MAX_EXPIRY, resource=None):
```

Comment:
FYI I don't we need this change from table to conf any longer nor the _cached_table var below. Changing the conf argument back will require a number of changes to test_db so I'm not going to cry over this